ONSdigital/census-rm-toolbox
# Msg persistence

# Motivation and Context
Pika doesn't set DeliveryMode to Persistent by default

# What has changed
set it to persistent

# How to test?
Locally I ran the ATs to get some data onto the db
Kill Case processor 'docker stop caseprocessor' 
Then harvested a valid caseId from the DB
Then in this branch of toolbox run: pipenv run python message_maker.py
Choose 'soft refusal'
Use the caseId and when asked to publish type 'yes'
Check rabbitmq case.refusals has a msg with persistence




# Links
